tyssue/core/sheet.py

```diff
@@ -91,6 +91,102 @@ def get_neighborhood(self, face, order):
 
         return neighbors.reset_index(drop=True).loc[1:]
 
+    def sheet_extract(self, key_word_column, coords=['x', 'y', 'z']):
+        """ Extract a new sheet from the embryo sheet
```

Comment:
maybe rename `key_word_column` as `face_mask` as 'keyword' is used for the named arguments in python, it's a bit confusing...
You also need to add it the docstring parameter list.